feat: Add contentDisposition option to file storages (#31974)
RocketChat/Rocket.Chat: .changeset/sweet-kiwis-scream.md

---
"@rocket.chat/meteor": minor
---

>  In the Desktop App, clicking on a message attachment link will now initiate a direct download of the attachment if not a PDF

Reviewer: ```suggestion
Clicking on a message attachment link on the Desktop App will now initiate a direct download of the attachment only when the attachment is not a PDF file
```
